### DISCUSSION TITLE
Setup guide for Svelte Kit

### DISCUSSION THREAD

**Discussion by Dhaiwat10** [tailwindlabs/tailwindcss#5503] [Ideas]
A guide for [Svelte Kit](https://kit.svelte.dev/) would be great! It's gaining quite a bit of traction among Svelte devs.

More than happy to put in a PR myself if that's fine! :)

**Comment by shajidhasan** [comment]
There's a package called [svelte-add](https://github.com/svelte-add/svelte-add). It works for me.
